### ISSUE TITLE
Can't click Mod Options

### ISSUE THREAD

**Issue by kuroyushi** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#89]
This might not be a problem with your mod but I have two mods that add menu options. AutoTrader and BannerlordTweaks.

This Mod also adds a menu, but at the bottom. I cannot click on Mod Options, but I can see it. I do not know if it is a mod issue or too many other third-party mods adding their own Mod Options menu.

[img]https://i.imgur.com/xP0Wkce.jpg[/img]

**Comment by kuroyushi** [comment]
I removed the mod called FreePlay and I can use the Mod Options now for your mod.

**Comment by Tyler-IN** [comment]
Thanks for reporting, I'll see about coming up with a new place to put something on the main menu.

**Comment by Eerye** [comment]
Sorry for butting in with a question I should probably ask somewhere else, but... Is it possible to make some kind of Mod Configuration Menu framework thingy, like the ones for Skyrim and Fallout 4? Which I think would eliminate this issue.

**Comment by Tyler-IN** [comment]
Yeah, we started breaking it off over here;
https://github.com/Tyler-IN/MnB2-Bannerlord-ModOptions

**Comment by Tyler-IN** [comment]
See also issue https://github.com/Tyler-IN/MnB2-Bannerlord-CommunityPatch/issues/61

**Comment by Eerye** [comment]
Oh didn't see those, thank you!

**Comment by Tyler-IN** [comment]
Until we get another options mechanism in place, the only way to access options there is to turn off some mods that have menus, load into the game w/ community patch enabled, access mod options and make changes, then exit and turn the other mods back on.

**Comment by imrikus** [comment]
Hello! For my part I have 4 mods enable along with the Community Patch, (AutoBlocker, Bannerlord Tweaks, AutoTrader, Mixed Gender Troops and the the CP). In these order they appear in the main menu: https://imgur.com/19buwVS
Having these enabled, so 5 more over the Vanilla clickable options that the main menu has you are not allowed to click from the Credits tab to the bottom, so I guess maybe the main menu can't accept a limit of 3 over the classic ones. If there isn't a way to make it accept more a first workaround was to make mixed gender do its work and later uninstall it and reactivate the other mods. Like that you can click all the options but it's a bit clunky... There are other mods that I don't know if it's alright to deactivate them as bannerlord tweaks or autoblock

**Comment by Eerye** [comment]
Yeah it's not ideal at all, I'm trying to use as few mods with an in-game options menu as I can until we have an MCM. And other mod authors basically using your own features against you is not cool either.

Also, @imrikus, if you read back you'll notice they are already working on a mod configuration framework.

**Comment by Tyler-IN** [comment]
Linking https://github.com/Tyler-IN/MnB2-Bannerlord-CommunityPatch/issues/130
